plugin/parser/differ/mysql/differ.go

```diff
@@ -45,7 +45,13 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to parse new statement %q", newStmt)
 	}
-	var diff []ast.Node
+
+	var add []ast.Node
+	var inplaceUpdate []ast.Node
+	var nonInplaceDrop [][]ast.Node
```

Comment:
`nonInplaceDrop` is not used

Reply:
Yeah, it's not used for now. But stay it here is ok for me. I will drop it if we still not use it after we accomplish the MVP.

Reply:
Add it when really used or comment to let use know it not used please

Reply:
Remove it.